features/vendor_bids_open_auction.feature

```diff
@@ -3,7 +3,15 @@ Feature: Vendor bids on an open auction
   I want to be able to bid on auctions
   So I can be paid for work
 
-  Scenario: Logging in before bidding
+  Scenario: Auction has no bids
+    Given there is an open bidless auction
+    And the auction has a start price of 1000
```

Comment:
needed it to not have it so we could set it as integer val -- could probably find a way to do it with dollar sign